SPECS/usrsctp/usrsctp.spec

```
@@ -0,0 +1,67 @@
+Summary:        A portable SCTP userland stack
+Name:           usrsctp
+Version:        0.9.5.0
+Release:        1%{?dist}
+License:        LGPLv2+ AND CC0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/sctplab/usrsctp
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz
```

Once the source tarball is re-named, you can make sure the spec finds the right file with this:
```suggestion
Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

---

Acknowledged. Spec file is updated in the recent commit.